spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/ConfigServiceTemplate.java

```diff
@@ -61,4 +92,15 @@
             throw new IllegalStateException(LOAD_FAILURE_MSG, e);
         }
     }
+
+    private static String getNextLink(CloseableHttpResponse response) {
+        // Extract next link path and query from response header
```

Comment:
not sure java doc is better if you want to comments the method.

Reply:
Updated Javadoc.